packages/dtogen/lib/application/core/type_string_extensions.dart

```diff
@@ -37,6 +39,14 @@ extension TypeStringExtensions on String {
     return replaceAll("Dto", "");
   }
 
+  String toCamelCase() {
+    return replaceAll(RegExp(r"(_|-)+"), " ")
+        .split(" ")
+        .map((str) => str.firstCharToUpperCase())
+        .join()
+        .firstCharToLowerCase();
```

Comment:
Does it makes sense? Previously you’ve called this method on the each part of this string

Reply:
It's to**Lower**Case :)